mkosi/qemu.py

```diff
@@ -712,6 +712,10 @@ def register_machine(config: Config, pid: int, fname: Path) -> None:
     )
 
 
+def available_cpus(config: Config) -> int:
+    return int(run(["nproc"], stdout=subprocess.PIPE, sandbox=config.sandbox()).stdout.strip())
```

Comment:
Is there anything `nproc` does differently from `os.cpu_count`? The latter calls `os.sched_getaffinity` and I would assume `nproc` does something similar.